sample/playableGame.py

```diff
@@ -0,0 +1,52 @@
+import sample.board as board
+board = board.Board()
+
+def get_next_sub_board():
+    if board.next_move == (-1,-1):
+        #get user entered subX,subY
+        print("You can choose which sub_board you want to play in next.")
+        nextSub = input("Enter the row and column separated by a comma : ")
+        nextSub = nextSub.split(',')
+        (subX,subY) = tuple((int(str) for str in nextSub))
+        #check that subX and subY are valid
+        while not board.is_valid(subX,subY):
+            nextSub = input("Please enter a valid row and column separated by a comma : ")
+            nextSub = nextSub.split(',')
+            (subX,subY) = tuple((int(str) for str in nextSub))
+    else:
```

Comment:
Since we do not want duplicated code you should create a function called something like get_user_coordinates()